app/assets/styles/_structure.scss

```diff
@@ -96,11 +96,26 @@
   .form__control {
     padding-right: 2.25rem;
   }
+
+  .autocomplete__menu-item {
+    padding: .5rem;
+    &.is-highlighted,
+    &:hover,
+    &:focus {
+      color: $link-color;
+      background-color: tint($link-color, 92%);
+      opacity: 1;
+    }
+  }
 }
 
 .global-search__button-go {
   @extend .button, .button--achromic, .button--text-hidden, .button--medium;
 
+  // TODO: Absolute positioning is not good
+  position: relative;
+  top: -3px;
```

Comment:
I don't like this, but I was taking so long figuring out a better way :/
Any ideas @danielfdsilva?